src/main/scala/is/hail/asm4s/CMHelp.scala

```diff
@@ -9,11 +9,11 @@ import scala.collection.mutable
 import scala.reflect.ClassTag
 
 // Without this, we can't use invokeStatic with CompilationHelp as a type.
-case class CompilationHelp() {}
+case class CMHelp() {}
```

Comment:
I don't understand this.  Non-companion object functions aren't static?  By CompilationHelp, do you mean CMHelp?